assets/styles/elements/button.css

```diff
@@ -36,6 +36,14 @@
     border: 0.0625rem solid var(--color-primary);
 }
 
+.button--scaled {
```

Comment:
`--scaled` is quite ambiguous. Scaled up or down? How much scaled? What about something like `button--bigger`?

Reply:
I think I misunderstood the name. Will add a comment and good.